lib/active_merchant/billing/gateways/elavon.rb

```diff
@@ -66,8 +76,10 @@ class ElavonGateway < Gateway
       # * <tt>:password</tt> -- PIN
       # * <tt>:user</tt> -- Specify a subuser of the account (optional)
       # * <tt>:test => +true+ or +false+</tt> -- Force test transactions
+      # * <tt>:custom_fields_array => custom fields configured on specific Elavon account</tt> (optional)
       def initialize(options = {})
         requires!(options, :login, :password)
+        @custom_fields_array = options[:custom_fields_array] || [:customer_number]
```

Comment:
`custom_fields_array` feels a bit off to me, how about just `custom_fields`? And how do you feel about moving the default to a constant?